diff --git i/codegra_fs/cgfs.py w/codegra_fs/cgfs.py
index 28998ea..7663107 100755
--- i/codegra_fs/cgfs.py
+++ w/codegra_fs/cgfs.py
@@ -38,8 +38,8 @@ if True:
     from codegra_fs.cgapi import CGAPI, APICodes, CGAPIException
 
 try:
-    import fuse  # type: ignore
-    from fuse import FUSE, Operations, FuseOSError, LoggingMixIn  # type: ignore
+    import fusepy  # type: ignore
+    from fusepy import FUSE, Operations, FuseOSError, LoggingMixIn  # type: ignore
     if sys.platform.startswith('win32'):
         import cffi
         import winfspy
diff --git i/codegra_fs/utils.py w/codegra_fs/utils.py
index a0f12fe..58067b3 100644
--- i/codegra_fs/utils.py
+++ w/codegra_fs/utils.py
@@ -26,7 +26,7 @@ def _get_fuse_version_info() -> t.Tuple[int, int]:
 
 def get_fuse_install_message() -> t.Optional[t.Tuple[str, t.Optional[str]]]:
     try:
-        import fuse  # type: ignore
+        import fusepy  # type: ignore
     except:
         pass
     else:
